hpat/tests/test_series.py

```diff
@@ -2234,6 +2242,63 @@ def test_impl(A, B):
         B = np.random.ranf(n)
         pd.testing.assert_series_equal(hpat_func(A, B), test_impl(A, B))
 
+    def test_series_sort_values_noidx(self):
+        def test_impl_true(S):
+            return S.sort_values(ascending=True)
+
+        def test_impl_false(S):
+            return S.sort_values(ascending=False)
+
+        hpat_func1 = hpat.jit(test_impl_true)
+        hpat_func2 = hpat.jit(test_impl_false)
+
+        data_test = [[6, 6, 2, 1, 3, 3, 2, 1, 2],
+                     [1.1, 0.3, 2.1, 1, 3, 0.3, 2.1, 1.1, 2.2],
+                     [6, 6.1, 2.2, 1, 3, 0, 2.2, 1, 2],
+                     [6, 6, 2, 1, 3, np.nan, np.nan, np.nan, np.nan],
+                     [3., 5.3, np.nan, np.nan, 33.2, 56.3, 4.4, 3.7, 8.9],
+                     ['a', 's', 'dd', 'm', 'll', '345', 'xrt', 'kd', 'qq'],
+                     ['dh', 'a', '', 'cv', 'b', '', 'b', 'b', 'p']
+                     ]
```

Comment:
You could use globally defined data as input data in tests: https://github.com/IntelPython/hpat/commit/f2435b037ecc403c6adada060b94279c6ac2ca31#diff-deca39d332649cea819383154a5d2cb3R39-R62